content/docs/1.7.0/advanced-resources/deploy/node-selector.md

```diff
@@ -55,7 +61,13 @@ You need to set node selector for both types of components. See more details bel
    * To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before applying it.
 
 2. Set node selector for user deployed components
-    * If you install Longhorn by Rancher UI, you need to click `Edit as YAML` and copy this values into the YAML then click upgrade:
+   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML to apply the values to all the components at once:
```

Comment:
```suggestion
   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (**SCREEN_TITLE** > **Edit as YAML**) to apply the value to all system-managed components.
```

Reply:
Can we specify the part of the Rancher UI where users can find the **Edit as YAML** button?